react/features/mobile/react-native-sdk/middleware.js

```diff
@@ -29,46 +30,49 @@ const externalAPIEnabled = isExternalAPIAvailable();
 
     switch (type) {
     case SET_AUDIO_MUTED:
-        rnSdkHandlers?.onAudioMutedChanged && rnSdkHandlers?.onAudioMutedChanged(action.muted);
+        rnSdkHandlers?.onAudioMutedChanged?.(action.muted);
         break;
     case SET_VIDEO_MUTED:
-        rnSdkHandlers?.onVideoMutedChanged && rnSdkHandlers?.onVideoMutedChanged(Boolean(action.muted));
+        rnSdkHandlers?.onVideoMutedChanged?.(Boolean(action.muted));
         break;
     case CONFERENCE_BLURRED:
-        rnSdkHandlers?.onConferenceBlurred && rnSdkHandlers?.onConferenceBlurred();
+        rnSdkHandlers?.onConferenceBlurred?.();
         break;
     case CONFERENCE_FOCUSED:
-        rnSdkHandlers?.onConferenceFocused && rnSdkHandlers?.onConferenceFocused();
+        rnSdkHandlers?.onConferenceFocused?.();
         break;
     case CONFERENCE_JOINED:
-        rnSdkHandlers?.onConferenceJoined && rnSdkHandlers?.onConferenceJoined();
+        rnSdkHandlers?.onConferenceJoined?.();
         break;
     case CONFERENCE_LEFT:
         //  Props are torn down at this point, perhaps need to leave this one out
         break;
     case CONFERENCE_WILL_JOIN:
-        rnSdkHandlers?.onConferenceWillJoin && rnSdkHandlers?.onConferenceWillJoin();
+        rnSdkHandlers?.onConferenceWillJoin?.();
         break;
     case ENTER_PICTURE_IN_PICTURE:
-        rnSdkHandlers?.onEnterPictureInPicture && rnSdkHandlers?.onEnterPictureInPicture();
+        rnSdkHandlers?.onEnterPictureInPicture?.();
+        break;
+    case ENDPOINT_MESSAGE_RECEIVED:
+        rnSdkHandlers?.onEndpointMessageReceived?.(action.data);
```

Comment:
Pass the sender too, since someone might be interested in it.

Reply:
Updated.